### PR TITLE
Feature/45: Extracted document code to virtual-schema-common-document repo

### LINKED ISSUE #45: Separate DynamoDB and document code

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #45

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -85,9 +85,21 @@
...
-            <version>${vscommon.version}</version>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.exasol</groupId>
+            <artifactId>virtual-schema-common-document</artifactId>
+            <version>${vs-common-document.version}</version>
+            <type>test-jar</type>
+            <scope>test</scope>
```
Comment (morazow) [inline pom.xml:96]: Could you please move this to below `<-- Test dependencies -->` or `<-- Integration test dependecies -->`?

File: pom.xml [hunk 2]
```diff
@@ -85,9 +85,21 @@
...
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.awssdk</groupId>
+            <artifactId>dynamodb</artifactId>
```
Comment (morazow) [inline pom.xml:100]: This does not require a version?

**Review by morazow — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (jakobbraun) [reply pom.xml:100]: no, the version is set for all AWS packages here https://github.com/exasol/dynamodb-virtual-schema/blob/cd028f74dfa1a83a28196609e1b76c9ba955ac7e/pom.xml#L54

Comment (morazow) [reply pom.xml:93]: :+1:

**Review by morazow — APPROVED** [review]

**Pull request merged**